app/hotels/src/allHotels/AllHotelsSearchRow.js

```diff
@@ -33,11 +36,13 @@ type PropsWithContext = {|
   +apiProvider: ApiProvider,
 |};
 
-class AllHotelsSearchRow extends React.Component<PropsWithContext> {
+class AllHotelsSearchRow extends React.Component<Props> {
   onGoToSingleHotel = () => {
     const hotelId = this.props.data.hotelId;
```

Comment:
I know it's not your change, but maybe
```suggestion
    const { hotelId } = this.props.data;
```